templates/bake/element/file_header.twig

```diff
@@ -0,0 +1,5 @@
+<?php
+declare(strict_types=1);
+
+namespace {{ namespace }};
+{{- Bake.concat("\n", [Bake.classUseStatements(imports.class ?? []), Bake.functionUseStatements(imports.function ?? []), Bake.constUseStatements(imports.const ?? [])], '\n\n') }}
```

Comment:
The individual use statements could be renders using an element instead of compiling with BakeHelper.

The same goes for imported methods.

Reply:
The question isn't customization but what's the best way to maintain this. I wasn't sure if writing code gen into BakeHelper was right.